Dockerfile-cache

```diff
@@ -0,0 +1,3 @@
+FROM alpine
```

Comment:
Please use a specific version instead of `latest`. Using the latest version will cause an unpredicted result.